bands/templates/bands/agreement-base.html

```diff
@@ -0,0 +1,181 @@
+        Performance time and date for {{ band.group.name }} to be determined, and will be provided prior to the event
+        as an update to this document.
+            <br>
+        Performer will be provided with <strong>{{ band.estimated_loadin_minutes }} minutes</strong>,
+        of set up and line-check, followed by <strong>{{ band.performance_minutes }} minutes</strong> for performance.
+            <br>
+        {% block band_text_timer %}
+        These time-frames will be enforced, via an on-stage timer, to ensure that performances occur in a timely fashion.
+        {% endblock band_text_timer %}
+    </p>
+{% endblock %}
+
+{% block band_text_merch %}
+    <h3>
+        Merch
+    </h3>
+    <p>
+        {% block band_text_merch_inner %}
+        Space will be provided in our marketplace area for merchandise sales; either at a dedicated table or at our
+        <b>Rock Island</b> sales area, where {{ c.EVENT_NAME }} staff will sell performer-provided merchandise throughout the festival.
+        It is recommended, but not required, for performers to use our Rock Island service in this area, as merchandise can be sold
+        more consistently, and for many more hours.  On the night of the performance, table space will also be provided immediately
+        outside the concert hall for merchandise sales and signings.
+        {% endblock band_text_merch_inner %}
+    </p>
+{% endblock %}
+
+{% block band_text_payment %}
+{% if band.payment %}
+    <h3>
+        Payment
+    </h3>
+    <p>
+        <strong>${{ band.payment }}</strong> by check on site after the show.
+    </p>
+    {% endif %}
+{% endblock %}
+
+{% block band_text_accommodations %}
+    <h3>
+        Accommodations
+    </h3>
+    <p>
+        {% block band_text_accommodations_inner %}
+        One hotel room with 2 queen beds will be provided,
+        with check in and out times provided as part of our final agreement.
+        {% endblock %}
+    </p>
+{% endblock %}
+
+{% block band_text_parking %}
+    <h3>
+        Parking
+    </h3>
+    <p>
+        {% block band_text_parking_inner %}
+        To be included with hotel room for <b>{{ band.vehicles }} vehicle{{ band.vehicles|pluralize }}</b>.
+        {% endblock %}
+    </p>
+{% endblock %}
+
+{% block band_text_admission %}
+    <h3>
+        Admission
+    </h3>
+    <p>
+        {% block band_text_admission_inner %}
+        Each performer will receive one badge, and one additional complementary badge for a guest.
+        Additional complimentary badges may be issued upon request.
+        {% endblock %}
+    </p>
+{% endblock %}
+
+{% block band_text_agreement_confirm %}
+    <h3>
+        Agreement
+    </h3>
+    <p>
+        By submitting your band's information<span class="printhide"> below</span>, you agree to the above terms.
+    </p>
+{% endblock %}
+
+{% block band_information_form %}
+<h3>
+    Your Information
+</h3>
+<form method="post" action="agreement" class="form-horizontal" role="form">
```

Comment:
@cybersenshi notes that this form here is actually the new styling and we need to put back in the old styling for this PR